docs/MODULES.md

```diff
@@ -3,118 +3,49 @@ Documentation of various Modules
 
 List of Module Documentations
 
-1.  [Crop](#crop-module)
-2.  [Segmented-Colormap](#segmented-colormap-module)
-3.  [FisheyeGl](#fisheyeGl-module)
-4.  [Add QR](#Add-QR-module)
-5.  [Average](#average-module)
-6.  [Blend](#blend-module)
-7.  [Blur](#blur-module)
-8.  [Brightness](#brightness-module)
-9.  [Channel](#channel-module)
-10. [Colorbar](#colorbar-module)
-11. [Colormap](#colormap-module)
-12. [Contrast](#contrast-module)
-13. [Convolution](#convolution-module)
-14. [DecodeQr](#decodeQr-module)
-15. [Dither](#dither-module)
-16. [DrawRectangle](#draw-rectangle-module)
-17. [Dynamic](#dynamic-module)
-18. [Edge-Detect](#edge-detect-module)
+1.  [Add QR](#Add-QR-module)
+2.  [Average](#average-module)
+3.  [Blend](#blend-module)
+4.  [Blur](#blur-module)
+5.  [Brightness](#brightness-module)
+6.  [Channel](#channel-module)
+7.  [Colorbar](#colorbar-module)
+8.  [Colormap](#colormap-module)
+9.  [ColorTemperature](#color-temperature)
+10. [Contrast](#contrast-module)
+11. [Convolution](#convolution-module)
+12. [Crop](#crop-module)
+13. [DecodeQr](#decodeQr-module)
+14. [Dither](#dither-module)
+15. [DrawRectangle](#draw-rectangle-module)
+16. [Dynamic](#dynamic-module)
+17. [Edge-Detect](#edge-detect-module)
+18. [FisheyeGl](#fisheyeGl-module)
 19. [FlipImage](#flipimage-module)
 20. [Gamma-Correction](#gamma-correction-module)
 21. [Gradient](#gradient-module)
-22. [Histogram](#histogram-module)
-23. [Import-image](#import-image-module)
-24. [Invert](#invert-module)
-25. [Ndvi](#ndvi-module)
-26. [Ndvi-Colormap](#ndvi-colormap-module)
-27. [Overlay](#overlay-module)
-28. [PaintBucket](#paint-bucket-module)
-29. [Resize](#resize-module)
-30. [ReplaceColor](#replacecolor-module)
-31. [Rotate](#rotate-module)
-32. [Saturation](#saturation-module)
-33. [Threshold](#threshold)
-34. [Tint](#tint)
-35. [ColorTemperature](#color-temperature)
-36. [Grid-Overlay](#grid-overlay)
-37. [MinifyImage](#minify-image)
+22. [Grid-Overlay](#grid-overlay)
+23. [Histogram](#histogram-module)
+24. [Import-image](#import-image-module)
+25. [Invert](#invert-module)
+26. [MinifyImage](#minify-image)
+27. [Ndvi](#ndvi-module)
+28. [Ndvi-Colormap](#ndvi-colormap-module)
+29. [Overlay](#overlay-module)
+30. [PaintBucket](#paint-bucket-module)
+31. [ReplaceColor](#replacecolor-module)
+32. [Resize](#resize-module)
+33. [Rotate](#rotate-module)
+34. [Saturation](#saturation-module)
+35. [Segmented-Colormap](#segmented-colormap-module)
+36. [Threshold](#threshold)
+37. [Tint](#tint)
 38. [Text-Overlay](#text-overlay)
```

Comment:
Hey your work is really great just a little change can you change the order for text overlay after segmented? Sorry for the trouble